Botkit/Microsoft.BotKit/Conversation/BotkitConversation.cs

```diff
@@ -129,81 +130,79 @@ public void Before(string threadName, Object obj)
         /// <summary>
         /// This private method is called before a thread begins, and causes any bound handler functions to be executed.
         /// </summary>
-        /// <param name="threadName">The thread about to begin</param>
-        /// <param name="dialogContext">The current DialogContext</param>
-        /// <param name="step">The current step object</param>
+        /// <param name="threadName">The thread about to begin.</param>
+        /// <param name="dialogContext">The current DialogContext.</param>
+        /// <param name="step">The current step object.</param>
         public async void RunBefore(string threadName, DialogContext dialogContext, IBotkitConversationStep step)
         {
-
         }
 
         /// <summary>
         /// Bind a function to run after the dialog has completed.
         /// The first parameter to the handler will include a hash of all variables set and values collected from the user during the conversation.
         /// The second parameter to the handler is a BotWorker object that can be used to start new dialogs or take other actions.
         /// </summary>
-        /// <param name="handler">In the form async(results, bot) { ... }</param>
+        /// <param name="handler">In the form async(results, bot) { ... }.</param>
         public async void After(Action<object, BotWorker> handler)
         {
-
         }
 
         /// <summary>
         /// This private method is called at the end of the conversation, and causes any bound handler functions to be executed.
         /// </summary>
-        /// <param name="context">The current dialog context</param>
-        /// <param name="any">An object containing the final results of the dialog</param>
-        public async void RunAfter(DialogContext context, object any)
```

Comment:
Please keep the **_async_** indicator. When this method gets implemented, it will need it.

Reply:
Done